torch/autograd/profiler.py

```diff
@@ -358,46 +362,67 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.profiling_end_time_ns = t0
 
         _run_on_profiler_stop()
+
+        self._stats.profiling_window_duration_sec = (
+            (self.profiling_end_time_ns - self.profiling_start_time_ns) * 1.0 / 1e9
+        )
+        if self.acc_events:
```

Comment:
Maybe it might be useful to add a comment in the description of this flag to state that we won't lazily get the events if it is present. It seems obvious to us but maybe not as much to the less acquainted? Can be in a follow up diff